buildspecs/core.feature

```diff
@@ -106,6 +106,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="module_util_core" value="true"/>
 		<flag id="module_verutil" value="true"/>
 		<flag id="module_vm" value="true"/>
+		<flag id="opt_criuSupport" value="true"/>
```

Comment:
I just want to confirm you want it enabled for all builds by default?

Reply:
no, that was not intentional

Reply:
It won't matter for openj9 builds which will adjust based on the `--with-criu-support` configure option, but it should probably be false so as not to affect internal IBM builds.